packages/framework/block-std/src/gfx/keyboard.ts

```diff
@@ -16,27 +16,35 @@ export class KeyboardController {
 
   private _init() {
     this._disposable.add(
-      this.std.event.add('keyDown', evt => {
-        const state = evt.get('keyboardState');
-
-        this.shiftKey$.value = state.raw.shiftKey && state.raw.key === 'Shift';
-        this.spaceKey$.value = state.raw.code === 'Space';
+      this._listenKeyboard('keydown', evt => {
+        this.shiftKey$.value = evt.shiftKey && evt.key === 'Shift';
+        this.spaceKey$.value = evt.code === 'Space';
       })
     );
 
     this._disposable.add(
-      this.std.event.add('keyUp', evt => {
-        const state = evt.get('keyboardState');
-
-        this.shiftKey$.value = state.raw.shiftKey && state.raw.key === 'Shift';
+      this._listenKeyboard('keyup', evt => {
+        this.shiftKey$.value =
+          evt.shiftKey && evt.key === 'Shift' ? true : false;
 
-        if (state.raw.code === 'Space') {
+        if (evt.code === 'Space') {
           this.spaceKey$.value = false;
         }
       })
     );
   }
 
+  private _listenKeyboard(
+    event: 'keydown' | 'keyup',
+    callback: (keyboardEvt: KeyboardEvent) => void
+  ) {
+    document.addEventListener(event, callback, false);
+
+    return () => {
+      document.removeEventListener(event, callback, false);
+    };
+  }
+
```

Comment:
Does listening on the document affect user interactions on the AFFiNE side?

Reply:
I'm not sure. It should be fine.

Reply:
Have you tested the situation where multiple editors exist at the same time?

Reply:
I've added an e2e test